account_payment_cancel/tests/test_payment_cancel.py

```diff
@@ -0,0 +1,145 @@
+from odoo.tests import TransactionCase
+from odoo.modules import get_module_resource
+
+
+class TestPaymentCancel(TransactionCase):
+
+    def setUp(self):
+        super(TestPaymentCancel, self).setUp()
+
+        self.invoice_name = 'test invoice pain000'
+        self.invoice_line_name = 'test invoice line pain000'
+        self.order_name = '2017/1013'
+        self.journal_name = '2017/1013'
+        self.payment_line_name = 'Ltest'
+
+        # Create payment order from the invoice
+        invoice = self.env['account.invoice'].search(
```

Comment:
I would prefer that you remove the yml data for the test and directly create your invoice from the code here.